packages/client/src/subscribe/OrderMessages.js

```diff
@@ -16,8 +16,9 @@ let ID = 0
  */
 
 export default function OrderMessages(client, options = {}) {
-    const { gapFillTimeout, retryResendAfter, maxGapRequests } = client.options
+    const { gapFillTimeout, retryResendAfter, maxGapRequests, orderMessages } = client.options
     const { streamId, streamPartition, gapFill = true } = validateOptions(options)
+    let enabled = !!(orderMessages && gapFill && maxGapRequests)
```

Comment:
About this condition. Do we differentiate between ordering and gap filling or is it an all on/off choice?

Reply:
no differentiation, but it may actually be useful to differentiate.